docs/user/tutorials/installing.md

```diff
@@ -1,11 +1,49 @@
 # Installing Hypper
 
-## From source (Linux, macOS)
+This guide shows how to install the Hypper CLI. Hypper can be installed either from source, or from pre-built binary releases.
+
+
+## From the Binary Releases (Linux, macOS, windows)
```

Comment:
```suggestion
## From the binary beleases (Linux, macOS, windows)
```

Reply:
(tried the GH "Add suggestion to batch".. sorry for the noise)

Reply:
done!